plugins/email_service/app/helpers/email_service/application_helper.rb

```diff
@@ -485,304 +1069,212 @@ def pretty_print_html(input)
-            identity_type: id_type
-          })
-          id = 0
-          ids.identities.each do |identity|
-            attrs = ses_client.get_identity_verification_attributes({
-              identities: [identity]
-            })
-            status = attrs.verification_attributes[identity].verification_status
-            token = attrs.verification_attributes[identity].verification_token
-            dkim_err, dkim_attr = get_dkim_attributes([identity])
-            if dkim_attr
-              dkim_enabled = dkim_attr[:dkim_attributes][identity][:dkim_enabled]
-              dkim_tokens = dkim_attr[:dkim_attributes][identity][:dkim_tokens]
-              dkim_verification_status = dkim_attr[:dkim_attributes][identity][:dkim_verification_status]
-            end
-            id += 1
-            identity_hash = {id: id, identity: identity, status: status,\
-            verification_token: token, dkim_enabled: dkim_enabled, \
-            dkim_tokens: dkim_tokens, dkim_verification_status: dkim_verification_status }
-            @verified_identities.push(identity_hash)
-          end
-        rescue Aws::SES::Errors::ServiceError => e
-          Rails.logger.error "error while listing verified identities. #{e.message}"
-        end
-        Rails.logger.info "Fetched identities are #{@verified_identities}"
-        return @verified_identities
-      end
-    end
-
-    # delete a verified identity
-    def remove_verified_identity(identity)
-
-      status = nil
-        begin
-          ses_client.delete_identity({
-            identity: identity
-          })
-          audit_logger.info(current_user.id, 'has removed verified identity ', identity)
-          status = "success"
-         rescue Aws::SES::Errors::ServiceError => e
-          status = "error: #{e.message}"
-          Rails.logger.error "error while removing verified identity. Error message: #{e.message}"
-        end
-
-        return status 
-    end
-
-    # list dkim attributes
-    def get_dkim_attributes(identities=[])
-
-      err = nil
-      dkim_attributes = {}
-
-      begin
-        dkim_attributes = ses_client.get_identity_dkim_attributes({
-          identities: identities, 
-        })
-      rescue Aws::SES::Errors::ServiceError => e
-        err = "#{e.message}"
-        Rails.logger.error "CRONUS: DEBUG: DKIM Attributes: #{err}"
-      end
-
-      return err, dkim_attributes
-
-    end
-
-    def get_dkim_tokens(resp, identity)
-      dkim_token = resp[:dkim_attributes][identity][:dkim_tokens]
-      return dkim_token
-    end
-    
-    def is_dkim_enabled(resp, identity)
-      dkim_enabled = resp[:dkim_attributes][identity][:dkim_enabled]
-    end
-    
-    def get_dkim_verification_status(resp, identity)
-      verification_status = resp[:dkim_attributes][identity][:dkim_verification_status] if resp
-
-      return verification_status if verification_status
-    end
-
-    def start_dkim_verification(identity)
+    # #
```

Comment:
Do you really want to keep all this? You have git to check how the file changed over the time.